packages/core/src/fhir-deduplication/__tests__/examples/condition-examples.ts

```diff
@@ -4,6 +4,11 @@ export const icd10CodeMd = {
   code: "H35.30",
   display: "Macular degeneration",
 };
+export const otherCodeSystemMd = {
```

Comment:
nit: IMO is international medical objects code system

Reply:
it's just being used as an example of something that isn't used for condition dedup, so ill leave it